src/constants.js

```diff
@@ -65,6 +65,8 @@ exports.REGISTER_PLUGIN_REPLY = 'registerPluginReply';
 exports.REMOVE_ACTION = 'removeAction';
 exports.REMOVE_THING = 'removeThing';
 exports.REQUEST_ACTION = 'requestAction';
+exports.METHOD_REJECTED = 'methodRejected';
```

Comment:
Since these are very specific to actions, can you make them something more appropriate, like `actionRequested` and `actionRejected`? You'll need to add another pair for removing actions.